pom.xml

```diff
@@ -94,7 +94,7 @@
         <ddf-branding>DDF</ddf-branding>
         <ddf-branding-lowercase>ddf</ddf-branding-lowercase>
         <ddf-branding-expanded>Distributed Data Framework</ddf-branding-expanded>
-        <ddf-version>${project.version}</ddf-version>
+        <ddf.version>${project.version}</ddf.version>
```

Comment:
Did you mean to change this to ddf.version instead of ddf-version. I think ddf-version is correct.